src/commodore/index.spec.ts

```diff
@@ -6,7 +6,11 @@ import {
   it,
   jest,
 } from '@jest/globals';
-import { mkdir } from 'fs/promises';
+import { copyFile, mkdir, readdir } from 'fs/promises';
+import { rmSync } from 'fs';
+
+import Git from 'simple-git';
```

Comment:
`simple-git` seems to be a dependency of renovate. I think we should add this as an explicit dependency.